ofApp.cpp

```diff
@@ -1,387 +0,0 @@
-        }
-        if (rightObj.colour != "right") {
-            font.drawString(wrapString(rightObj.jsonStringName, 550, true), 650, 65);
-        }
-        ofSetColor(255); // reset colour back to white - likely dont need this as it is also at the start of the draw function but may cause issues when using keys to move between screens if removed
-    }
-    else if (screen == screens::game_over) {
-        backgroundImage.draw(0, 0, 1200, 750); // this should cover the whole game over screen at all times
-        metalGearSolid.draw(300, 200, 600, 300); // SNAKEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEEE - should have looked to see if i could put videos or sound in omds
-        gameOverTextAreaButton.set(300, 200, 600, 300); // this button takes you back to the main menu
-
-        font.drawString(ofToString(leftObj.score), 450, 100); // score is the score that you just accumulated 
-        font.drawString("Score:", 300, 100);
-
-        font.drawString(ofToString(leftObj.highscore), 525, 150); // highscore is the best score you have acheived since running the app. will reset when app closes
-        font.drawString("Highscore:", 300, 150);
-    }
-    else if (screen == screens::land) { // after adding this i looked up how many countries are landlocked vs arent and theres way more which arent so this is like a free life every time - still works tho 
-        gameOverTextAreaImage.draw(0, 0, 1200, 750); // this should cover the whole land screen at all times - changed to use the gameOver image here bc i wanted the yellow even tho this isnt the proper way of doing thatbla bla bla
-        ofSetColor(0);
-        font.drawString("Ah, you got that wrong!", 50, 100); // L
-        font2.drawString("But you'll get a second chance if you can say whether or not:", 50, 200);
-        font2.drawString(ofToString(leftObj.jsonStringName), 680, 200);
-        font2.drawString("is landlocked. Press 'T' for true, 'F' for false.", 50, 250); // not sure if there is an easier way of doing this - i couldnt figure one out so this text is split across 3 lines
-        
-        //ofDrawBitmapString(leftObj.landlocked, 50, 500); //YzxNkP99T9WYb8
-    
-    }
-    else if (screen == screens::none) {
-        cout << "loading";
-    }
-    else { // when the user clicks on the land screen it runs this but it doesnt break the code or slow it down or anything so its staying.
-        cout << "Something has gone horribly wrong - but dont worry it doesnt matter :)" << endl;
-    }
-}
-
-void ofApp::keyPressed(int key){
-    // code which should run when the user has to pick whether a country is landlocked or not 
-    if (screen == screens::land && (key == 'T' || key == 't')) {
-        //user thinks its landlocked
-        if (leftObj.landlocked == "true") {
-            //correct
-            screen = screens::game;
-        }
-        else {
-            screen = screens::game_over;
```

Comment:
Does the user only have one life before game over?

Reply:
Could implement a lives system allowing for different difficulty levels to enhance the user experience, a new state / screen with 3 buttons for difficulty which could assign an amount of lives to a variable.